definitions/consented/json/CaseEvent/CaseEvent.json

```diff
@@ -699,8 +699,8 @@
     "Name": "Awaiting Information",
     "Description": "Awaiting Information",
     "DisplayOrder": 38,
-    "PreConditionState(s)": "applicationSubmitted;applicationIssued;responseReceived;orderMade;consentOrderApproved;infoReceived",
-    "PostConditionState": "awaitingInfo",
```

Comment:
PostConditionState change? Does this change need?

Reply:
Hi Rishi, well spotted, I think I changed this from 'awaitingInformation', to '*', because the 'Add Note', event was not there, but I'll revert the change back, and re-test again. Thanks Rishi.

Reply:
they are asking to change post condition.

Reply:
Yes, they are under the AC2:-
AC2. Add events to Awaiting Information state
GIVEN I am logged in a caseworker on a consented paper case 
AND the case is in Awaiting Information state 
WHEN I select the Next Steps menu 
THEN I should see the following events available:
Add Note
AND I should be able to complete these events and progress the case.
Note: all these events currently exist in consented FR and are available to caseworkers.

Reply:
It does not say that you need to make it *.

Reply:
That was the only way to reveal the 'Add Note', event because if the PostConditionState = awaitingInfo, and you run that event, then on the next step drop down list, there is no  'Add Note',thanks

Reply:
No.